internal/core/graphics/color.rs

```diff
@@ -305,12 +305,7 @@ impl Color {
 
 impl InterpolatedPropertyValue for Color {
     fn interpolate(&self, target_value: &Self, t: f32) -> Self {
-        Self {
-            red: self.red.interpolate(&target_value.red, t),
-            green: self.green.interpolate(&target_value.green, t),
-            blue: self.blue.interpolate(&target_value.blue, t),
-            alpha: self.alpha.interpolate(&target_value.alpha, t),
-        }
+        self.mix(target_value, t)
```

Comment:
Actually, I checked, and it seems like this should be reversed:

```suggestion
        target_value.mix(self, t)
```

(The mix argument order is a bit misleading)

Reply:
Fixed this. The test I added initially didn't catch it since it only checked the middle point for mixing =)